scripts/erc20_to_erc20/sendForeign.js

```diff
@@ -30,6 +31,8 @@ async function main() {
   const poa20 = new web3Foreign.eth.Contract(ERC20_ABI, ERC20_TOKEN_ADDRESS)
 
   try {
+    await isValidAmount(FOREIGN_MIN_AMOUNT_PER_TX, bridge)
```

Comment:
the minimum amount per transaction is not applicable for the Foreign side of the bridge in `erc20-to-erc20` mode.

Reply:
Fixed